linux_os/guide/system/permissions/files/file_permissions_var_log_stig/tests/excluded_files.pass.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+find /var/log -exec chmod g-rwx,o-rwx {} \;
+
+{{% if product in ['ubuntu2004', 'ubuntu2204'] %}}
```

Comment:
As the scenario is designed only for Ubuntu I think it should be marked with the `# platform` header.